### PR TITLE
use org.apache.pekko as org id

### PR DESCRIPTION
**Pull request by pjfanning**
relates to #34

### CODE REVIEW

**Comment by He-Pin** [comment]
any ETA for the first milestone cut？

**Comment by He-Pin** [comment]
> > any ETA for the first milestone cut？
> 
> I'm afraid I don't know - there is an awful lot to do - I would be surprised if we got a milestone done in 2022 at the current rate of progress

I see, holiday is coming too

**Comment by spangaer** [comment]
Naïve backwards planning.
1.0.0 March 7 2023, 6 months ahead of the security fix expiry date. Allowing consumers to get their feat wet.
Jan or Feb would be better, but I doubt that to be realistic.
0.5.0 Mid January, runs renamed, but has known issues.
0.9.0 Mid Feb, RC-ish

Is that crazytalk?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -256,7 +256,7 @@ validate the binary compatibility of incoming pull requests. If your PR fails du
 an error like this:
 
 ```
-[info] akka-stream: found 1 potential binary incompatibilities while checking against com.typesafe.akka:akka-stream_2.12:2.4.2  (filtered 222)
+[info] akka-stream: found 1 potential binary incompatibilities while checking against org.apache.pekko:akka-stream_2.12:2.4.2  (filtered 222)
```
Comment (Claudenw) [inline CONTRIBUTING.md:259]: should akka-stream be pekko-stream

File: akka-docs/src/main/paradox/actors.md [hunk 2]
```diff
@@ -7,13 +7,13 @@
...
-  bomGroup=com.typesafe.akka bomArtifact=akka-bom_$scala.binary.version$ bomVersionSymbols=AkkaVersion
+  bomGroup=org.apache.pekko bomArtifact=akka-bom_$scala.binary.version$ bomVersionSymbols=AkkaVersion
   symbol1=AkkaVersion
   value1="$akka.version$"
-  group="com.typesafe.akka"
+  group="org.apache.pekko"
   artifact="akka-actor_$scala.binary.version$"
   version=AkkaVersion
-  group2="com.typesafe.akka"
+  group2="org.apache.pekko"
```
Comment (Claudenw) [inline akka-docs/src/main/paradox/actors.md:16]: Don;'t the artifact names have to change from "akka-" to "pekko-"

**Review by Claudenw — COMMENTED** [review]
It seems like this is only 1/2 done.  Are the symbols actually AkkaVersion or should they be PekkoVersion?

The directory name is akka-docs but should that not be pekko-docs?

File: akka-docs/src/main/paradox/actors.md (hunk 2 above)
Comment (pjfanning) [reply akka-docs/src/main/paradox/actors.md:16]: I'm between a rock and a hard place. If I do many PRs with smaller changes, people might review them. If I do all the changes in one PR, most people will refuse to review it.

https://github.com/apache/incubator-pekko/pull/20 renames some of the modules. Another PR or 2 like #20 and all the modules will be renamed.

**Comment by mdedetrich** [comment]
@Claudenw In regard to your comments I would also prefer splitting this up over various PR's rather than doing too much at once. This way its easier to track if something goes wrong

File: akka-docs/src/main/paradox/actors.md (hunk 2 above)
Comment (pazkooda) [reply akka-docs/src/main/paradox/actors.md:16]: PR title says you're handling _groupId_. It's already "huge" change in terms of files but easy to review due to properly narrowed scope of it.

One step at at time :)

Comment (mdedetrich) [reply akka-docs/src/main/paradox/actors.md:16]: After working on my PR regarding renaming the package, I kind of under-estimated the scope that some of these changes require. I have come to the conclusion that we might have to deal with some massive PR's, at least if we want to uphold the "no PR shall break CI build" (which I think there isn't any argument against).

File: CONTRIBUTING.md (hunk 1 above)
Comment (pjfanning) [reply CONTRIBUTING.md:259]: This has not yet been renamed to akka-stream - I have a different PR open for that

File: akka-docs/src/main/paradox/actors.md (hunk 2 above)
Comment (pjfanning) [reply akka-docs/src/main/paradox/actors.md:16]: @Claudenw I'd like to see if I can get this merged. Do you think that this PR is problematic?

**Comment by pjfanning** [comment]
@mdedetrich would you have time to review this?

**Comment by mdedetrich** [comment]
@pjfanning Will do, ill try to get it done by end of week.

**Comment by mdedetrich** [comment]
@pjfanning PR needs to be rebased. Otherwise its lgtm, will recheck and approve after the rebase to quickly see if anything has been missed.

**Comment by pjfanning** [comment]
@mdedetrich I'll merge this after the rebuild completes (if it's clean)

**Comment by mdedetrich** [comment]
> @mdedetrich I'll merge this after the rebuild completes (if it's clean)

No worries

**Pull request merged**